build.gradle

```diff
@@ -48,7 +48,8 @@ subprojects {
         compile 'org.slf4j:slf4j-api:1.6.1'
         compile 'org.slf4j:slf4j-log4j12:1.6.1'
 
-
+        // Archaius-2
+        compile group: 'com.netflix.archaius', name: 'archaius2-core', version: '2.1.11'
```

Comment:
I think that is what I started with but gradle changed it to this version. I will give it a try again